doc/data/messages/t/too-complex/good.py

```diff
@@ -1 +1,36 @@
-# This is a placeholder for correct code for this message.
+fruit_prices = {
+    "apple": 1.1,
+    "pear": 0.8,
+    "banana": 1.2,
+    "mango": 3.5,
+    "peach": 0.5,
+    "melon": 4.9,
+    "orange": 2.0,
+    "strawberry": 2.5,
+    "mandarin": 2.3,
+    "plum": 0.5,
+    "watermelon": 6.4,
+}
+
+
+def get_price(fruit):
+    full_price = fruit_prices.get(fruit)
+    if fruit in fruits_to_buy:
+        return (float(full_price)) * 50 / 100
```

Comment:
```suggestion
def fifty_percent_off(whole):
    return (float(whole)) * 50 / 100

def get_price(fruit):
    full_price = fruit_prices.get(fruit)
    if fruit in discounted_fruits:
        return fifty_percent_off(full_price)
```

Reply:
I integrated the discount into the `get_price` function because i thought you wanted to get rid of the `fifty_percent_off` function and the `discounted_fruits` list.

Reply:
Sorry it's hard to suggest thing properly, I probably messed up somewhere 😄